pydynamo_brain/pydynamo_brain/ui/radiiPainter.py

```diff
@@ -210,3 +215,13 @@ def zoomedLocation(self, xyz):
     # HACK - utilities
     def isNearZ(self, z):
         return abs(z - self.zAt) < 3
+
+    #Given a selected point and an additional neightboring point
+    #returns a point perpendicular to the line joining these points
+    def caliperPoints(self, origin, point, ifNext):
+        x, y, _ = util.normDelta(point.location, origin.location)
```

Comment:
As discussed, this might not go too well when the Z delta is big.
Best to take **just** the X & Y delta, norm that, then create vector as you do here,
which will guarantee the result is also unit length.

If this is done, I'd recommend also adding that it returns a vector of length 1.